README.md

```diff
@@ -7,12 +7,39 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
 
 ## How to use
 
+
 ### Install
 
 ```
 npm install loopback-connector-couchbasex --save
 ```
 
+### Notice
+
+- Model's function `update` play the same role as `updateAttributes` which cause by `loopback-datasource-juggler`.
+
+`PUT` and `PATCH` both do the same thing that merge incoming data with current record.
+
+ ```
+ update = Object.assign(current, incoming)
+ ```
+
+Discussion of `update` and `updateAttribetes` , see: [https://groups.google.com/forum/#!topic/loopbackjs/-1jarvOuh8k](https://groups.google.com/forum/#!topic/loopbackjs/-1jarvOuh8k)
+
+- since version 1.0.0 the default vaule of  `stale` was set to `ok` in viewQuery.
+
+If the update condition field isn't `id`  you must put `stale` in `options` parameter to make sure you get the correct data, like:
```

Comment:
```suggestion
If the update condition field isn't `id`  you might need to set `stale` in `options` parameter to make sure you get the latest data if you have a strong need of data instant consistency, like:
```

Reply:
I think `stale` must be set to 1 in a write operation. Otherwise, the dirty read and dirty write would happen.

Reply:
k, I got you, you are talking about `write`, LGTM.